introduction.md

```diff
@@ -0,0 +1,2 @@
+user.name = jinyaoyao

```

Comment:
I have already given some repairment and pushed up. But I still cannot pass the 'Collaboration Using Pull Requests' on codeschool.